addOns/ascanrulesBeta/CHANGELOG.md

```diff
@@ -19,6 +19,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
     - Session Fixation
     - Cross-Domain Misconfiguration 
 - Add website alert links to the help page (Issue 8189).
+- Remove broken http link for alerts (Issue 8262):
+    - Possible Username Enumeration
+    - Out of Band XSS
+    - Cookie Slack Detector
```

Comment:
Should only cover what was actually changed.

Reply:
resolved in the latest commit